src/DynamoCore/DynamoCore.csproj

```diff
@@ -29,7 +29,7 @@
     <PackageReference Include="Autodesk.IDSDK" Version="1.1.6" />
     <PackageReference Include="Greg" Version="3.0.0.2955" />
     <PackageReference Include="DynamoVisualProgramming.LibG_229_0_0" Version="3.0.0.2855" GeneratePathProperty="true" />
```

Comment:
Is there a newer version of LibG_229 as well? On second thoughts, how about completely removing it?

Reply:
I don't know if I want to remove it just yet. Many devs a probably relying on 229 still.
I will check if there is a newer 229 version.